api/src/main/resources/workbench-api.yaml

```diff
@@ -9421,6 +9460,7 @@ components:
       enum:
       - ACTIVE
       - INACTIVE
+      - EXPIRED
```

Comment:
I'm not sure about adding another enum type here.  The exhausted-credits logic sets this to INACTIVE, and we've got a fair amount of logic that depends on this. 
 I have been thinking of expired-credits as following the exhausted behavior as close as possible, so I'd like to think through the pros/cons of differing in this respect.